### PR TITLE
Commend section

### PR DESCRIPTION
**Pull request by EviNikolaidou**

### CODE REVIEW

File: app/services/CommentSectionSerivces.scala [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package services
+
+import models.Commends
+
+class CommentSectionSerivces {
```
Comment (RafSobolQAC) [inline app/services/CommentSectionSerivces.scala]: Codacy Issue found: [Redundant braces after class definition](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325&bid=17278877)

File: app/services/RatingServives.scala [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package services
+
+class RatingServives {
```
Comment (RafSobolQAC) [inline app/services/RatingServives.scala]: Codacy Issue found: [Redundant braces after class definition](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)

File: app/controllers/CommendSectionController.scala [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  def serviceSubmitComment(comment: Commends) = {
+    collection.flatMap(_.insert.one(comment))
+  }
+
+  def submitCommend = Action.async { implicit request: Request[AnyContent] =>
+    Commends.createCommentForm.bindFromRequest.fold({ formWithErrors =>
+      println(formWithErrors)
+      Future.successful(BadRequest(views.html.commends(formWithErrors)))
+    }, { Commends =>
```
Comment (RafSobolQAC) [inline app/controllers/CommendSectionController.scala]: Codacy Issue found: [Field name 'Commends' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)

File: app/controllers/CommendSectionController.scala [hunk 4]
```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import akka.http.scaladsl.model.headers.LinkParams.title
+import javax.inject.Inject
+import models.Commends
+import play.api.mvc._
+import play.modules.reactivemongo.{MongoController, ReactiveMongoApi, ReactiveMongoComponents}
+import services.ReleasedServices
```
Comment (RafSobolQAC) [inline app/controllers/CommendSectionController.scala]: Codacy Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325&bid=17278877)

File: app/controllers/CommendSectionController.scala [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+  def serviceSubmitComment(comment: Commends) = {
+    collection.flatMap(_.insert.one(comment))
+  }
+
+  def submitCommend = Action.async { implicit request: Request[AnyContent] =>
+    Commends.createCommentForm.bindFromRequest.fold({ formWithErrors =>
+      println(formWithErrors)
+      Future.successful(BadRequest(views.html.commends(formWithErrors)))
+    }, { Commends =>
```
Comment (RafSobolQAC) [inline app/controllers/CommendSectionController.scala]: Codacy Issue found: [Field name 'Commends' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325&bid=17278877)

File: app/controllers/CommendSectionController.scala [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+  def serviceSubmitComment(comment: Commends) = {
+    collection.flatMap(_.insert.one(comment))
+  }
+
+  def submitCommend = Action.async { implicit request: Request[AnyContent] =>
+    Commends.createCommentForm.bindFromRequest.fold({ formWithErrors =>
+      println(formWithErrors)
+      Future.successful(BadRequest(views.html.commends(formWithErrors)))
+    }, {Commends =>
```
Comment (RafSobolQAC) [inline app/controllers/CommendSectionController.scala]: Codacy Issue found: [Field name 'Commends' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)

Comment (RafSobolQAC) [reply app/controllers/CommendSectionController.scala]: camelCase for variables

File: app/controllers/CommendSectionController.scala [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+                                          val releasedServices: ReleasedServices
+                                        ) extends AbstractController(components)
+  with MongoController with ReactiveMongoComponents with play.api.i18n.I18nSupport {
+  implicit def ec: ExecutionContext = components.executionContext
+
+  def collection: Future[JSONCollection] = reactiveMongoApi.database.map(_.collection[JSONCollection]("comment"))
+
+
+  def getCommends = Action { implicit request: Request[AnyContent] =>
+    Ok(views.html.commends(Commends.createCommentForm))
```
Comment (malikbensalem) [inline app/controllers/CommendSectionController.scala:25]: not a big deal but it's "comments" not "commends"

File: app/controllers/CommendSectionController.scala [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+
+  def serviceSubmitComment(comment: Commends) = {
+    collection.flatMap(_.insert.one(comment))
+  }
+
+  def submitCommend = Action.async { implicit request: Request[AnyContent] =>
+    Commends.createCommentForm.bindFromRequest.fold({ formWithErrors =>
+      println(formWithErrors)
```
Comment (malikbensalem) [inline app/controllers/CommendSectionController.scala:35]: dont need "println(formWithErrors)"

File: app/models/MessageBoard.scala [hunk 9]
```diff
@@ -0,0 +1,21 @@
+//package models
```
Comment (malikbensalem) [inline app/models/MessageBoard.scala:1]: do you need this file?

File: app/models/Commends.scala [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+                         name: String,
+                         email: String,
+                         movieName: String,
+                         comment: String
+                       )
+
+object Commends {
+  val createCommentForm: Form[Commends] = Form(
+    mapping(
+      "named" -> nonEmptyText,
```
Comment (malikbensalem) [inline app/models/Commends.scala:16]: instead of "named" did you mean "names" or "name"?

**Review by malikbensalem — COMMENTED** [review]
Few spelling errors?

**Review by malikbensalem — APPROVED** [review]
Nice

**Pull request merged**